config/schemas/form_outline.json

```diff
@@ -0,0 +1,118 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "Coronavirus form outline",
+    "type": "object",
+    "properties": {
+      "medical-equipment": {
+        "type": "array",
+        "items": {
+          "type": "string",
+          "enum": [
+            "/medical-equipment",
+            "/are-you-a-manufacturer",
+            "/medical-equipment-type",
+            "/product-details",
+            "/testing-equipment",
+            "/additional-product"
+          ]
+        }
+      },
+      "accommodation": {
+        "type": "array",
+        "items": {
+          "type": "string",
+          "enum": [
+            "/accommodation",
+            "/rooms-number"
+          ]
+        }
+      },
+      "offer-transport": {
+        "type": "array",
+        "items": {
+          "type": "string",
+          "enum": [
+            "/offer-transport",
+            "/transport-type"
+          ]
+        }
+      },
+      "offer-space": {
+        "type": "array",
+        "items": {
+          "type": "string",
+          "enum": [
+            "/offer-space",
+            "/offer-space-type"
+          ]
+        }
+      },
+      "offer-staff": {
+        "type": "array",
+        "items": {
+          "type": "string",
+          "enum": [
+            "/offer-staff",
+            "/offer-staff-type"
+          ]
+        }
+      },
+      "expert-advice-type": {
+        "type": "array",
+        "items": {
+          "type": "string",
+          "enum": [
+            "/expert-advice-type",
+            "/construction-services",
+            "/it-services"
+          ]
+        }
+      },
+      "offer-care": {
+        "type": "array",
+        "items": {
+          "type": "string",
+          "enum": [
+            "/offer-care",
+            "/offer-care-qualifications"
+          ]
+        }
+      },
+      "offer-other-support": {
+        "type": "array",
+        "items": {
+          "type": "string",
+          "enum": [
+            "/offer-other-support"
+          ]
+        }
+      },
+      "location": {
+        "type": "array",
+        "items": {
+          "type": "string",
+          "enum": [
+            "/location"
+          ]
+        }
+      },
+      "business-details": {
+        "type": "array",
+        "items": {
+          "type": "string",
+          "enum": [
+            "/business-details"
+          ]
+        }
+      },
+      "contact-details": {
+        "type": "array",
+        "items": {
+          "type": "string",
+          "enum": [
+            "/contact-details"
+          ]
+        }
+      }
+    }
+  }
```

Comment:
Minor, we're missing a new blank line at eof.

Reply:
Thanks @alex-ju 👍   resolved